config/locales/en/helpers.yml

```diff
@@ -407,3 +409,4 @@ en:
         adult_suspended_prison_sentence: *order_started_lead_text
         adult_disqualification: ''
         youth_disqualification: ''
+        
```

Comment:
This file is missing the newline at the end of the file. 
You should be able to setup your editor to automatically add it if it's missing when you save a file.
It can also be setup to remove any unnecessary whitespace.